src/lib/settings.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path')
```

Comment:
I've started a new concept of `libs` instead of `utils` that we currently have.
It seems that most of our current `utils` are not used for sharing code, but just to extract functionality from the different commands.
However some commands/utils need to share the same code - so I started putting those parts under lib.